### ISSUE TITLE
Oxygen-oxygen violates energy conservation

### ISSUE THREAD

**Issue by arouinfar** [phetsims/atomic-interactions#56]
Observed in dev.11. I believe @jbphet and I have talked about this before, but I couldn't track down the relevant issue to reopen.

If the unpinned oxygen is released somewhere within the well, it sometimes jumps out of the well. This violates energy conservation. This could be a legacy bug, since I've seen it in the Java version too.

I also have a concern that the oxygen settles to the bottom of the well. Classically, I would expect the atom to oscillate to the same height on both sides of the well, so we're somehow losing energy in the system. I also know that oxygen is a bit different from the other pairs, since it's naturally diatomic. I'm probably overlooking something here, so can you take a look at this @amyh-phet?

Here are two screencasts, both in slow motion.
![oxygenhtml5](https://cloud.githubusercontent.com/assets/8419308/17633284/b6533d28-6089-11e6-9c56-9ed4e90ebcbd.gif)
![oxygenjava](https://cloud.githubusercontent.com/assets/8419308/17633285/b656ca92-6089-11e6-8e34-1b727f7048f4.gif)

**Comment by amyh-phet** [comment]
@arouinfar. Happy to look into this.  Likely won't t get back to you until Monday.  Let me know if you need an answer sooner.

**Comment by arouinfar** [comment]
Monday should be fine @amyh-phet!

I know that jumping out of the well definitely violates energy conservation, but it's less clear to me if settling at the bottom of the well is also problematic.

**Comment by jbphet** [comment]
If I recall correctly, settling at the bottom of the well was part of the design.  This is why the pinned atom vibrates - as a way to indicate that the energy is going somewhere.  In real life, I believe the energy often goes into the kinetic energy of the newly bonded pair of atoms.

The teacher tip document says, "When the bond forms, the shaking of the left atom is meant to
represent the energy transferred away from the bonded pair."

**Comment by amyh-phet** [comment]
@jbphet  and @arouinfar    I'm okay with the behavior that leads to settling at the bottom of the well.   I do think we need to address the the issue of the atoms being able to move outside the potential energy when the unpinned atom is released inside the well.  I agree that it is an issue with a violation of the law of conservation of energy.  I would expect that the atom could not travel up the potential energy curve higher than the point it was released.

**Comment by arouinfar** [comment]
Thanks for taking a look at this @amyh-phet!

> This is why the pinned atom vibrates - as a way to indicate that the energy is going somewhere.

This was the critical detail I was overlooking @jbphet!  Settling at the bottom of the well makes sense if I consider the energy lost through the vibration of the pinned atom.

It looks like the real issue is that oxygen violates energy conservation by occasionally jumping above its release point outside of the well.

**Comment by arouinfar** [comment]
We should make a note about the pinned oxygen vibrating in the teacher tips for SOM and AI, so I'm tagging this with teaching resources and self-assigning.

**Comment by arouinfar** [comment]
Correction -- oxygen-oxygen only appears in AI, so I'll leave this one to @amyh-phet since she'll be working on the teacher tips.

**Comment by jbphet** [comment]
This was caused by the vibration of the fixed atom during bonding - it was vibrating randomly, and could thus end up getting closer to the movable atom, thus creating a high repulsive potential.  I've adjusted the vibration pattern of the fixed atom during the bonding phase, and that seems to have resolved the issue.  Assigning to @arouinfar to verify on master and see if the current behavior looks good.
